botfront/values.yaml

```diff
@@ -124,4 +124,18 @@ mongo-express:
       - host:
         paths:
           - /
-    tls: []
\ No newline at end of file
+    tls: []
+
+redis: 
+  # Set enabled: true to deploy redis
+  enabled: false
+  # redis is used as a lock database for rasa, so async update between master and slaves can create issues
+  cluster:
+    enabled: false
```

Comment:
This probably where the risks of changing them should be explained (rather than the readme)